CONTRIBUTING.md

```diff
@@ -1,3 +1,32 @@
 # Contributing
 
 We follow the [IPython Contributing Guide](https://github.com/ipython/ipython/blob/master/CONTRIBUTING.md).
+
+## To set up a development environment
+
+Fork the repository and clone the forked repository locally.
+
+Use Conda to install dependencies and activate the development environment.
+
+```
+conda create -n qtdev python=3
+conda activate qtdev
+conda install pyqt
+```
+
+To run after the changes have been made to source (preferred):
```

Comment:
Please add blank below this line

Reply:
@ccordoba12 Did you mean after " To run after the changes have been made to source (preferred): "?

Reply:
Yep, that's what I meant.

Reply:
Added the line and reworded. Let me know if this is what you wanted.